measures/ResStockArguments/measure.rb

```diff
@@ -740,7 +733,7 @@ def run(model, runner, user_arguments)
       rescue
       end
 
-      if args_to_delete.include? arg_name
+      if args_to_delete.include?(arg_name) || (arg_value == Constants.Auto)
```

Comment:
And then at the very end, if there is a value of Constants.Auto, we exclude it from being passed to the BuildResHPXML measure (i.e., we don't provide a value for the optional arguments, which means they get defaulted).